geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListMembersCommand.java

```diff
@@ -74,4 +80,18 @@ public Result listMember(@CliOption(key = {CliStrings.GROUP}, unspecifiedDefault
     }
     return result;
   }
+
+  private String getCoordinator() {
+    String result = "unknown";
+    InternalDistributedSystem ids = InternalDistributedSystem.getConnectedInstance();
+    if ((ids != null) && (ids.isConnected())) {
+      MembershipManager mmgr = ids.getDistributionManager().getMembershipManager();
+      DistributedMember coord = ((GMSMembershipManager) mmgr).getCoordinator();
```

Comment:
This code shouldn't refer to GMSMembershipManager.  Why don't you add a getCoordinator() method to NetView or to the MembershipManager interface?

Reply:
i added in interface MembershipManager.